lamby/tests/models/test_user.py

```diff
@@ -0,0 +1,21 @@
+from sqlalchemy import exc
+
+from lamby.models.user import User
+from lamby.database import db
+
+
+def test_create_user(app):
```

Comment:
Can we add test data destruction? Like after each test the db shouldn't be changed.